src/definitions/modules/tab.less

```diff
@@ -25,6 +25,9 @@
 .ui.tab {
   display: none;
 }
+.ui.tab > .item {
```

Comment:
By design the tab items are only set within menus not within the tab itself (which just holds the content).
There also is not other `.item` setting in `tab.less`  and there is no docs example 
That said, i think this is not neccessary and could be removed.

Reply:
Fixed in
https://github.com/fomantic/Fomantic-UI/pull/638/commits/3ababe893b478eb4f2ea0182213221197eb32c4b

Removed `.ui.tab. > .item`.

And applied changes to not only `.ui.tabular.menu .item`  but every `.ui.menu .item` since `div.item` may be used in vertical menu or other menuvariation.

Reply:
For generic menus there's already a .link.item variation which makes them hoverable. If every .ui.menu .item gets  a pointer cursor, doesn't it make .link.item useless?

Reply:
@donaggio Good catch! I'll add a comment to the other codepart of the PR